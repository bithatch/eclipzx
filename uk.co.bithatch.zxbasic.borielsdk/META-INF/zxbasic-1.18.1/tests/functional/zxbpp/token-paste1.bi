

#define aa(x) (AA x)

#define x(a, b, c) a##b(c)

x(a, a, 1)
