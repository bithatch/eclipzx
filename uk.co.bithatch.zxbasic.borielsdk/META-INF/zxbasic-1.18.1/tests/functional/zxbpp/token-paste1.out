#line 1 "token-paste1.bi"


#line 4 "token-paste1.bi"

#line 6 "token-paste1.bi"

(AA 1)
